code/scripting/scripting.h

```diff
@@ -152,10 +152,9 @@ struct script_action
 
 class ConditionedHook
 {
-private:
+public:
```

Comment:
Why is this necessary to be public now? I don't see a reason in the code to change that.

Reply:
Because AssayActions needs to be able to iterate over the Actions vector, but it isn't a ConditionedHook member function. Conditions doesn't strictly need to be private but I felt it was cleaner just to take the honest route and make the whole thing public.

Reply:
Ok, that is a valid point. In that case, please just make it a `struct` and remove the `public` altogether.

Reply:
The general convention throughout the codebase is that, if it has methods, it's a `class`.

Reply:
Alright I guess.